src/app/wizard/wizard.component.html

```diff
@@ -39,20 +39,20 @@ <h3 class="blank-slate-pf-main-action">{{config?.loadingTitle}}</h3>
 </div>
 <div class="modal-footer wizard-pf-footer pfng-wizard-position-override"
      [ngClass]="{'pfng-footer-inline': config?.embedInPage}">
-  <button class="btn btn-default btn-cancel wizard-pf-cancel" type="button"
-          [ngClass]="{'wizard-pf-cancel-no-back': config?.hidePreviousButton}"
+  <button class="btn btn-default wizard-btn btn-cancel" type="button"
           [disabled]="config?.done"
           (click)="cancel()"
           *ngIf="!config?.embedInPage">{{config?.cancelTitle}}</button>
-  <button class="btn btn-default" type="button"
+  <button class="btn btn-default pfng-wizard-previous-btn" type="button"
           tooltip="{{selectedStep?.config?.previousTooltip}}" placement="left"
+          [ngClass]="{'pfng-wizard-btn-no-back': config?.hidePreviousButton}"
           [disabled]="!config?.ready || config?.done || !selectedStep?.previousEnabled || firstStep"
           (click)="previous(true)">{{config?.previousTitle}}</button>
   <button class="btn btn-primary wizard-pf-next" type="button"
           tooltip="{{selectedStep?.config?.nextTooltip}}" placement="left"
           [disabled]="!config?.ready || !selectedStep?.nextEnabled"
           (click)="next(true)">{{config?.nextTitle}}</button>
-  <button class="btn btn-default btn-cancel wizard-pf-cancel pfng-cancel-inline" type="button"
+  <button class="btn btn-default btn-cancel wizard-btn-cancel pfng-cancel-inline" type="button"
```

Comment:
Just curious why the rename to wizard-btn-cancel? I don't believe wizard-btn-cancel or wizard-pf-cancel actually exist in patternfly-ng or patternfly. We can just remove it?